package.json

```diff
@@ -8,7 +8,7 @@
     "url": "https://github.com/angular/material2.git"
   },
   "scripts": {
-    "build": "gulp build:release",
+    "build": "gulp material:build-release:clean",
```

Comment:
Yeah I see that it might be nicer when building different packages on their own. I personally vote for organizing the tasks by target. 

It just feels not very consistent if you want to copy for example the assets of the demo-app.

- e.g `copy-assets:demo-app`. 

I would be fine revisiting later in another PR or issue.